### PR TITLE
Settings and Paragraph sections

### PR DESCRIPTION
**Pull request by ElenaGallo**
## Description

Updated the Story Settings section and the Paragraph Section

## Other useful information
Read the Docs [link](https://elena-mapstore-docs.readthedocs.io/en/5886_userguide/user-guide/story-setting/)

### CODE REVIEW

File: docs/user-guide/paragraph-section.md [hunk 1]
```diff
@@ -13,15 +13,19 @@ The Paragraph Section allows to insert a textual content to the story. The story
...
-With a click on the text area, it is possible to write the text and customize it through the [Text Editor Toolbar](text-editor-toolbar.md).  A possible result of adding and formatting the text can be the following: 
+* Change the size of the text content: clicking on the **Change Size** button <img src="../img/button/change_size2.jpg" class="ms-docbutton"/> a dropdown menu appears to allow selecting between *Small*, *Medium* or *Full* size: 
 
-<img src="../img/paragraph-section/p-text-content.jpg" class="ms-docimage"/>
+<img src="../img/paragraph-section/paragraphe-size.jpg" class="ms-docimage"/>
 
-The user can delete the text content with the Remove button  <img src="../img/button/remove2.jpg" class="ms-docbutton"/>.
+* Delete the *Paragraph  Section* through the **Remove** button <img src="../img/button/remove2.jpg" class="ms-docbutton"/>
+
+The editor can write a test clicking on the text content and customize it through the [Text Editor Toolbar](text-editor-toolbar.md). A possible result of adding and formatting the text can be the following:
```
Comment (MV88) [inline docs/user-guide/paragraph-section.md:26]:
```suggestion
The editor can write a text by clicking on the text content and customize it through the [Text Editor Toolbar](text-editor-toolbar.md). A possible result of adding and formatting the text can be the following:
```

**Review by MV88 — CHANGES_REQUESTED** [review]
in paragraph section there is a button image that is not loading
image


by looking at some big images in the doc, i see them as blurred, not nitid. Is it just me?

File: docs/user-guide/story-setting.md [hunk 2]
```diff
@@ -1,29 +1,43 @@
...
+* The **Theme** to choose the default background and text color and the default font of the whole story: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color: 
 
 <img src="../img/story-setting/theme.gif" class="ms-docimage"/>
 
-* The **Overlay** to choose the default background and text color of the *Contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content) as well: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
+ * The *Font Family* to choose the default text font present in the whole story: clicking on the search bar a dropdown menu opens to allow selecting the desired font (*Inherit*, *Arial*, *Georgia*, *Impact*, *Tahoma*, *Time New Roman*, *Verdana*):
```
Comment (tdipisa) [inline docs/user-guide/story-setting.md:15]: We should put a note here (with a link to the related configration in dev guide) to notify that the list of available fonts families can be customized by configuration file.

File: docs/user-guide/story-setting.md [hunk 3]
```diff
@@ -1,29 +1,43 @@
...
 
 <img src="../img/story-setting/theme.gif" class="ms-docimage"/>
 
-* The **Overlay** to choose the default background and text color of the *Contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content) as well: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
+ * The *Font Family* to choose the default text font present in the whole story: clicking on the search bar a dropdown menu opens to allow selecting the desired font (*Inherit*, *Arial*, *Georgia*, *Impact*, *Tahoma*, *Time New Roman*, *Verdana*): 
+
+<img src="../img/story-setting/font_family.gif" class="ms-docimage"/>
+
+* The **Overlay** to choose the default background and text color of the *contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content) as well: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
```
Comment (tdipisa) [inline docs/user-guide/story-setting.md:19]: * The **Overlay** to choose the default background and text color of overlay *contents* ...

Use a better image for the color picker button

File: docs/user-guide/story-setting.md [hunk 4]
```diff
@@ -1,29 +1,43 @@
...
 <img src="../img/story-setting/story-setting-1.jpg" class="ms-docimage" style="max-width:400px"/>
 
-The editor can enable or disable and edit the following components:
+## Story Theme
 
-* The **Title** of the story, the default value is the title given to the story's resource in MapStore
+The editor can customize the different components of the story through the following sections:
 
-* The **Theme** to choose the default background and text color of the whole story: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color: 
+* The **Theme** to choose the default background and text color and the default font of the whole story: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
```
Comment (tdipisa) [inline docs/user-guide/story-setting.md:11]: Use a better image for the color picker button

File: docs/user-guide/story-setting.md [hunk 5]
```diff
@@ -1,29 +1,43 @@
...
+ * The *Font Family* to choose the default text font present in the whole story: clicking on the search bar a dropdown menu opens to allow selecting the desired font (*Inherit*, *Arial*, *Georgia*, *Impact*, *Tahoma*, *Time New Roman*, *Verdana*): 
+
+<img src="../img/story-setting/font_family.gif" class="ms-docimage"/>
+
+* The **Overlay** to choose the default background and text color of the *contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content) as well: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
 
 <img src="../img/story-setting/overlay.gif" class="ms-docimage"/>
 
-* The **Shadow** of the *Content* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content):
+* The *Shadow* of the *contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content): enabling or disabling the shadow:
```
Comment (tdipisa) [inline docs/user-guide/story-setting.md:23]: * The **Shadow** of overlay *contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content): enabling or disabling the shadow

File: docs/user-guide/story-setting.md [hunk 6]
```diff
@@ -1,29 +1,43 @@
...
 
 <img src="../img/story-setting/shadow.gif" class="ms-docimage"/>
 
+* The **Links** to choose the default color of the hyperlink that may be present in the *Text Contents*: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
```
Comment (tdipisa) [inline docs/user-guide/story-setting.md:27]: * The color of **Links**. The story editor can choose the default color of the hyperlink that may be present in the *Text Contents*: clicking on the *Change Color* button  a color picker appears to allow selecting the desired color.

Use a better image for the color picker button

File: docs/user-guide/story-setting.md [hunk 7]
```diff
@@ -1,29 +1,43 @@
...
+
+## Story Header
+
+In the top bar of the story, if the editor enables them, the following components are added:
+
+<img src="../img/story-setting/story_header.jpg" class="ms-docimage" width="400px"/>
+
+* The **Title** of the story, the default value is the title given to the story's resource in MapStore.
+
+* The *Font Size* of the title: clicking on the search bar a dropdown menu opens to allow selecting the desired size (*14 px*, *16 px*, *18 px*, *24 px*, *28 px*).
```
Comment (tdipisa) [inline docs/user-guide/story-setting.md:39]: The **Font Size** of the title: ...

**Review by tdipisa — CHANGES_REQUESTED** [review]

**Comment by ElenaGallo** [comment]
@tdipisa in reference to [this](https://github.com/geosolutions-it/MapStore2/pull/5906#pullrequestreview-493402727) comment, if the images of the Story Settings section and the Paragraph Section are now fine I open a new issue to change the blurred images in all the docu

File: docs/user-guide/story-setting.md [hunk 8]
```diff
@@ -1,29 +1,43 @@
...
 
-* The **Overlay** to choose the default background and text color of the *Contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content) as well: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
+* The **Overlay** to choose the default background and text color of overlay *contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content) as well: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
 
 <img src="../img/story-setting/overlay.gif" class="ms-docimage"/>
 
-* The **Shadow** of the *Content* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content):
+* The *Shadow* of overlay *contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content): enabling or disabling the shadow:
```
Comment (tdipisa) [inline docs/user-guide/story-setting.md:24]: Remove the final :

It is better the following at the end:

... to enable or disable the shadow.

File: docs/user-guide/story-setting.md [hunk 9]
```diff
@@ -1,29 +1,43 @@
...
 
 <img src="../img/story-setting/shadow.gif" class="ms-docimage"/>
 
+* The color of **Links**. The story editor can choose the default color of the hyperlink that may be present in the *Text Contents*: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
```
Comment (tdipisa) [inline docs/user-guide/story-setting.md:27]: remove the final : 

...color picker appears to allow selecting the desired color.

**Review by tdipisa — CHANGES_REQUESTED** [review]

**Comment by tdipisa** [comment]
@ElenaGallo I've reviewed it without looking at a final build in RtD: the doc of this PR in RtD is not yet updated. Please do that for the next review.

**Review by tdipisa — CHANGES_REQUESTED** [review]
Did you update the build in RdD? If yes, this image mentioned by @MV88 [here](https://github.com/geosolutions-it/MapStore2/pull/5906#pullrequestreview-493402727) is still broken @ElenaGallo.

**Comment by ElenaGallo** [comment]
@tdipisa The RdD has been updated and the broken icon has been fixed with [this](https://github.com/geosolutions-it/MapStore2/issues/5948) PR

**Pull request merged**